woocommerce/woocommerce
# [Experimental] Add styling controls

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/51787

This PR adds additional styling to the active filter block per design.

- Added Orientation and Justification to the toolbar and inspector controls of the Removable Chips block as discussed p1730901995741429-slack-C05UMTA7W0P
- Added Dimensions and Border controls to the parent Active Filters block.
- We decided to not include the block spacing for now as discussed p1730901995741429-slack-C05UMTA7W0P
- I've decided to not include Typography to the parent Active Filters block because each inner blocks actually has their own Typography controls and some of them are even custom. Both Removable chips and Clear button has their own. This is my own opinion though ref: p1730901995741429-slack-C05UMTA7W0P



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

> [!NOTE]  
> **Pre-requisite**
> - If you're using WC Beta Tester plugin, make sure the `experimental-blocks` feature is enabled (Tools > WCA Test Helper > Features).
> - Make sure to reset Product Catalog Template.

1. Navigate to Product Catalog template (Appearance > Editor > Templates > Product Catalog template.
2. Edit the template.
3. With the block inserter, add Product Filters (Experimental) block to the template.
4. Open the Document Overview (List view) and ensure you see the Product Filter Active block in the new design structure.
5. Select the parent Active Filters block
6. Ensure you see Dimensions and Border controls in the inspector controls.
7. Ensure each of those settings work in the editor and frontend.
8. Select the Removable Chips block.
9. Ensure you see the Orientation (Vertical, Horizontal) settings in the toolbar.
10. Go to the Inspector Controls -> Settings Panel and ensure you also see the Orientation settings there.
11. Try toggling vertical and horizontal settings and watch it change in the editor. Save and view it in the frontend. It should reflect the same as your saved settings.
12. The clear button setting has already been tested in prior PR https://github.com/woocommerce/woocommerce/pull/52268



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 
Adds additional styling settings to active filter block
</details>
